driver/ppm_events_public.h

```diff
@@ -1572,7 +1572,7 @@ enum extra_event_prog_code
 	PPM_SC_X(MEMFD_SECRET, 343) \
 	PPM_SC_X(IOPERM, 344) \
 	PPM_SC_X(KEXEC_FILE_LOAD, 345) \
-	PPM_SC_X(PIDFD_GET_FD, 346) \
```

Comment:
This was unfortunately a wrong name, since the syscall is called `pidfd_getfd`.
Renamed to the correct one, and dropped the one that was added by syscalls-bumper.